examples/cloud-operations/multi-cluster-mesh-gke-fleet-api/README.md

```diff
@@ -0,0 +1,65 @@
+# Multi-cluster mesh on GKE (fleet API)
+
+The following example shows how to create a multi-cluster mesh for two private clusters on GKE that are. Anthos Service Mesh with automatic control plane management is set up for clusters using the Fleet API. This can only be done is the clusters are in a single project and in the same VPC. In this particular case both clusters having being deployed to different subnets in a a shared VPC. 
```

Comment:
> The following example shows how to create a multi-cluster mesh for two private clusters on GKE that are

That are what?

> This can only be done is
This can only be done **if**

> In this particular case both clusters having being deployed to different subnets in a a shared VPC.
"a a"